cmd/runtimetest/main.go

```diff
@@ -191,6 +191,20 @@ func validateSysctls(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateROPaths(spec *rspec.Spec) error {
+	fmt.Println("validating readonlyPaths")
+	for _, v := range spec.Linux.ReadonlyPaths {
+		fi, err := os.Stat(v)
```

Comment:
And it's probably worth making the “is this path readonly?” a separate function so it can be shared with the check in #81.